spec/models/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
```

Comment:
- [ ] Good job on testing model validation but, Kindly include tests for all methods in your models as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.).